website/docs/developer/development-guide/code/integration-tests.md

```diff
@@ -161,7 +161,7 @@ This is a list of tests that should be created for existing Orca architecture. T
      :::
   1. Call the OrcaCopyToGlacierWorkflow to ingest the granules to Orca.
      :::tip
-     Make sure to cover ExcludeFileTypes being set, being unset, and excluding/allowing proper files in either case.
+     Make sure to cover excludeFileTypes being set, being unset, and excluding/allowing proper files in either case.
```

Comment:
`excludeFileTypes` should be `excludedFileExtensions`

Reply:
added